clients/drcachesim/tests/record_filter_unit_tests.cpp

```diff
@@ -427,6 +430,110 @@ test_encodings2regdeps_filter()
     return true;
 }
 
+/* Test SYS_futex preservation.
+ */
+#define SYS_futex_test 202
```

Comment:
These could be constexpr ints inside the function.  That avoids affecting subsequent tests down below and keeps it local.

Reply:
Done.